src/backend/distributed/citus--8.2-2--8.2-3.sql

```diff
@@ -0,0 +1,3 @@
+/* citus--8.2-2--8.2-3.sql */
+
+DROP FUNCTION IF EXISTS pg_catalog.citus_server_id();
```

Comment:
We also need to drop `pg_dist_node_metadata`, but currently few other tools (docker, cloud, etc.) depend on existence of that. We will remove it after updating those.